api/client/builder/types_test.go

```diff
@@ -666,25 +667,21 @@ var testExampleExecutionPayloadDenebTooManyBlobs = fmt.Sprintf(`{
       "proofs": [
         "0xb4021b0de10f743893d4f71e1bf830c019e832958efd6795baf2f83b8699a9eccc5dc99015d8d4d8ec370d0cc333c06a"
       ],
-      "blobs": [
-        "%s",
-        "%s",
-        "%s",
-        "%s",
-        "%s",
-        "%s",
-        "%s"
-      ]
+      "blobs": %s
     }
   }
-}`, hexutil.Encode(make([]byte, fieldparams.BlobLength)), // 1
-	hexutil.Encode(make([]byte, fieldparams.BlobLength)), // 2
-	hexutil.Encode(make([]byte, fieldparams.BlobLength)), // 3
-	hexutil.Encode(make([]byte, fieldparams.BlobLength)), // 4
-	hexutil.Encode(make([]byte, fieldparams.BlobLength)), // 5
-	hexutil.Encode(make([]byte, fieldparams.BlobLength)), // 6
-	hexutil.Encode(make([]byte, fieldparams.BlobLength)), // 7
-)
+}`, morethanMaxBlobs())
+
+func morethanMaxBlobs() string {
```

Comment:
not sure if this function is worth it... it's pretty slow, wondering if moving to beacon config so i can override might work better

Reply:
Hmm yeah, I imagine that would be quite slow. It's generating a 1 GiB string!

```
128kb per blob * 4096 blobs * 2 bytes per hex value = 1073741824 bytes
```

Reply:
the error should be triggered if it's empty so i'll do that